From c4a5433f0de337ff71d3a153b5b1d3132b5e09ec Mon Sep 17 00:00:00 2001
From: Max Merchel <Max.Merchel@tq-group.com>
Date: Fri, 18 Jan 2019 09:45:05 +0100
Subject: [PATCH 066/188] input: keyboard: gpio_keys: add missing header
 consumer.h

fix [-Werror=implicit-function-declaration]
of function 'pinctrl_pm_select_sleep_state' and
of function 'pinctrl_pm_select_default_state'

Signed-off-by: Max Merchel <Max.Merchel@tq-group.com>
---
 drivers/input/keyboard/gpio_keys.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/input/keyboard/gpio_keys.c b/drivers/input/keyboard/gpio_keys.c
index a3b2311474d8..2b064920a61c 100644
--- a/drivers/input/keyboard/gpio_keys.c
+++ b/drivers/input/keyboard/gpio_keys.c
@@ -29,6 +29,7 @@
 #include <linux/gpio/consumer.h>
 #include <linux/of.h>
 #include <linux/of_irq.h>
+#include <linux/pinctrl/consumer.h>
 #include <linux/spinlock.h>
 
 struct gpio_button_data {
